### PR TITLE
fix(explorer): fixing explorer db conn flag 

### PR DESCRIPTION
**Pull request by DanFlannel**
indexer and graphql services weren't reading the `explorer-db-conn` flag, this fixes that 🦾 

task: none

### CODE REVIEW

File: e2e/docker/compose.yaml.tmpl [hunk 1]
```diff
@@ -142,10 +142,8 @@ services:
       e2e: true
     container_name: explorer_indexer
     image: omniops/explorer-indexer:{{or .OmniTag "main"}}
-    {{- if .ExplorerMockDB}}
     command:
-      - --db-url=postgres://omni:password@explorer_db:5432/omni_db
-    {{- end}}
+      - --db-url={{ .ExplorerDBConn }}
```
Comment (pavelnikolov) [inline e2e/docker/compose.yaml.tmpl:146]: This should be `explorer-db-conn` as per above changes.

File: explorer/graphql/app/config.go [hunk 2]
```diff
@@ -15,7 +15,7 @@ import (
 
 const (
 	defaultListenAddr = ":8080"
-	defaultDBURL      = "postgres://omni:password@db:5432/omni_db"
+	defaultDBURL      = ""
```
Comment (pavelnikolov) [inline explorer/graphql/app/config.go:18]: This constant is not needed at all.

File: explorer/indexer/app/config.go [hunk 3]
```diff
@@ -14,8 +14,8 @@ import (
 )
 
 const (
-	defaultDBURL       = "postgres://omni:password@db:5432/omni_db"
-	defaultNetworkFile = "network.json"
+	defaultDBURL       = ""
```
Comment (pavelnikolov) [inline explorer/indexer/app/config.go:17]: same

File: explorer/indexer/app/testdata/default_explorer_indexer.toml [hunk 4]
```diff
@@ -10,10 +10,10 @@ version = "unknown"
 #######################################################################
 
 # URL at which the postgres database can be reached.
-dburl = "postgres://omni:password@db:5432/omni_db"
+dburl = ""
```
Comment (pavelnikolov) [inline explorer/indexer/app/testdata/default_explorer_indexer.toml:13]: Old variable name

File: e2e/vmcompose/testdata/TestSetup_127_0_0_6_compose.yaml.golden [hunk 5]
```diff
@@ -25,6 +27,8 @@ services:
       e2e: true
     container_name: explorer_graphql
     image: omniops/explorer-graphql:7d1ae53
+    command:
+      - --db-url=
```
Comment (pavelnikolov) [inline e2e/vmcompose/testdata/TestSetup_127_0_0_6_compose.yaml.golden:31]: old variable name

File: e2e/docker/testdata/TestComposeTemplate_main_explorer.golden [hunk 6]
```diff
@@ -157,6 +159,8 @@ services:
       e2e: true
     container_name: explorer_graphql
     image: omniops/explorer-graphql:main
+    command:
+      - --db-url=fake_connection
```
Comment (pavelnikolov) [inline e2e/docker/testdata/TestComposeTemplate_main_explorer.golden:163]: old var name

File: lib/cmd/flags.go [hunk 7]
```diff
@@ -42,11 +42,13 @@ func LogFlags(ctx context.Context, flags *pflag.FlagSet) error {
...
-// passwords in valid URLs as well as flags that contains words like "token" or "password" or "secret".
+// passwords in valid URLs as well as flags that contains words like "token", "password", "secret", "db" or "key".
 func redact(flag, val string) string {
 	if strings.Contains(flag, "token") ||
 		strings.Contains(flag, "password") ||
-		strings.Contains(flag, "secret") {
+		strings.Contains(flag, "secret") ||
+		strings.Contains(flag, "db") ||
+		strings.Contains(flag, "key") {
 		return "xxxxx"
```
Comment (DanFlannel) [inline lib/cmd/flags.go:52]: This is bad, definitely higher priority to fix in a follow up

**Review by pavelnikolov — APPROVED** [review]

**Pull request merged**